src/goto-instrument/full_slicer.cpp

```diff
@@ -399,6 +399,8 @@ void full_slicert::operator()(
       const exprt &s=to_code_dead(e_it->first->code).symbol();
       decl_dead[to_symbol_expr(s).get_identifier()].push(e_it->second);
     }
+    else if(e_it->first->is_function_call())
+      add_to_queue(queue, e_it->second, e_it->first);
```

Comment:
Add a comment that this assumes that a rechability slice has been performed, for it would otherwise add way too much to the slice.